Content.Server/GameObjects/Components/Items/FloorTileItemComponent.cs

```diff
@@ -21,12 +22,15 @@ public class FloorTileItemComponent : Component, IAfterInteract
         [Dependency] private readonly IMapManager _mapManager = default!;
 
         public override string Name => "FloorTile";
-        private string _outputTile;
+        private List<string> _outputTiles;
+        private int _default;
+
 
         public override void ExposeData(ObjectSerializer serializer)
         {
             base.ExposeData(serializer);
-            serializer.DataField(ref _outputTile, "output", "floor_steel");
+            serializer.DataField(ref _outputTiles, "outputs", null);
+            serializer.DataField(ref _default, "default", 0);
```

Comment:
You should just take the first tile in the list as the default, not do this ugly hack.